alfresco-transform-core-aio/alfresco-transform-core-aio/src/test/java/org/alfresco/transformer/AIOTransformRegistryTest.java

```diff
@@ -298,7 +297,7 @@ private void transformTextAndCheck(String text, String encoding, String checkTex
         parameters.put(PAGE_LIMIT, pageLimit);
         parameters.put(TRANSFORM_NAME_PARAMETER, "textToPdf");
         Transformer transformer = aioTransformerRegistry.getByTransformName("textToPdf");
-        transformer.transform(sourceFile, targetFile, "text/plain", "application/pdf", parameters);
+        transformer.transform("text/plain", "application/pdf", parameters, sourceFile, targetFile);
```

Comment:
Why change the order on this method ? Will it affect custom T-Engines (or AIS) ?

Reply:
It should be okay. If not there will be a compile error. The order is now the same in all the methods rather than a mixture.